spec/CampMinder/EstablishConnection_spec.rb

```diff
@@ -31,24 +30,24 @@
       expect(@establish_connection).not_to be nil
     end
 
-    it 'assigns the clientID attribute' do
-      expect(@establish_connection.clientID).to eq @data['clientID']
+    it 'assigns the client_id attribute' do
+      expect(@establish_connection.client_id).to eq @data['client_id']
     end
 
-    it 'assigns the personID attribute' do
-      expect(@establish_connection.personID).to eq @data['personID']
+    it 'assigns the person_id attribute' do
+      expect(@establish_connection.person_id).to eq @data['person_id']
     end
 
     it 'assigns the token attribute' do
       expect(@establish_connection.token).to eq @data['token']
     end
 
-    it 'assigns the partnerClientID attribute' do
-      expect(@establish_connection.partnerClientID).to eq @data['partnerClientID']
+    it 'assigns the partner_client_id attribute' do
+      expect(@establish_connection.partner_client_id).to eq @data['partner_client_id']
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.